brave/browser-laptop
# Using RenderFrameHost::CopyImageAt for image copy

fix #7388

Auditors: @bbondy, @bsclifton

Test Plan:
1. Go to https://www.brave.com/about/images/brave_icon_512x.png
2. Click "Copy Image" from context menu
3. The image should be copied to your clipboard instead of url
4. Paste it to somewhere else like slack channel to verify

## Submitter Checklist:


## Test Plan:


## Reviewer Checklist:


Tests
